frameworks/JavaScript/hyperexpress/app.js

```diff
@@ -0,0 +1,134 @@
+import { escape } from 'html-escaper'
+import { Server } from 'hyper-express'
+import { LRUCache } from 'lru-cache'
+import { isWorker, worker } from 'node:cluster'
+import { maxQuery, maxRows, message, json, extra } from './config.js'
+const { DATABASE } = process.env
+const db = DATABASE ? await import(`./database/${DATABASE}.js`) : null
+
+const generateRandomNumber = () => Math.ceil(Math.random() * maxRows)
+
+const parseQueries = (i) => Math.min(Math.max(parseInt(i, 10) || 1, 1), maxQuery)
+
+const cache = new LRUCache({
+  max: maxRows
+})
+
+const app = new Server()
+
+// use middleware to add `Server` into response header
+app.use((_request, response, next) => {
+  response.header('Server', 'hyperexpress')
+  next()
+})
+
+app.get('/plaintext', (_request, response) => {
+  response.atomic(() => {
+    response
+      .type('text')
+      .send(message)
+  })
+})
+
+app.get('/json', (_request, response) => {
+  response.json(json)
```

Comment:
> i. For each request, an object mapping the key message to Hello, World! must be instantiated.

Reply:
> > i. For each request, an object mapping the key message to Hello, World! must be instantiated.

Hi @nbrady-techempower , thanks for the review, I have updated the code according to your instruction.